gtk4/src/custom_filter.rs

```diff
@@ -0,0 +1,82 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::CustomFilter;
+use glib::object::IsA;
+use glib::translate::*;
+use std::ptr;
+
+pub trait CustomFilterExtManual: 'static {
+    fn new<F>(filter_func: F) -> CustomFilter
```

Comment:
The same thing as CustomSorter , this should be moved to `impl CustomSorter` directly